### PR TITLE
Image import: Always run inspection

### PR DESCRIPTION
**Pull request by EricEdens**
This removes the flag `-inspect` from `gce_vm_image_import`. It was a temporary, non-advertised flag that we used while shadow testing OS and disk inspection.

### CODE REVIEW

File: cli_tools/common/disk/inspect.go [hunk 1]
```diff
@@ -36,11 +35,14 @@ const (
 	workflowFile = "image_import/inspection/boot-inspect.wf.json"
 )
 
+// To rebuild the mock for DaisyWorker, run `go generate ./...`
```
Comment (zoran15) [inline cli_tools/common/disk/inspect.go:38]: Inspector, not DaisyWorker?

File: cli_tools_tests/module/diskinspect/disk_inspect_test.go [hunk 2]
```diff
@@ -228,7 +228,7 @@ func TestInspectDisk(t *testing.T) {
 			imageURI: "projects/compute-image-tools-test/global/images/windows-2019",
 			expected: &pb.InspectionResults{
 				OsRelease: &pb.OsRelease{
-					CliFormatted: "windows-2019-x64",
```
Comment (zoran15) [inline cli_tools_tests/module/diskinspect/disk_inspect_test.go:231]: Was there a bug earlier (regarding the extra "-x64") that was not detected?

File: cli_tools/common/disk/inspect.go (hunk 1 above)
Comment (EricEdens) [reply cli_tools/common/disk/inspect.go]: done

File: cli_tools_tests/module/diskinspect/disk_inspect_test.go (hunk 2 above)
Comment (EricEdens) [reply cli_tools_tests/module/diskinspect/disk_inspect_test.go:231]: The tests were stale, and I didn't see them until running them on this PR. (I'm still trying to figure out automation for the tests in `cli_tools_tests` directory. I created b/176264612 to fix this.)

**Review by zoran15 — APPROVED** [review]

File: cli_tools/common/disk/inspect_test.go [hunk 3]
```diff
@@ -35,13 +34,12 @@ import (
 
 func TestBootInspector_Inspect_PassesVarsWhenInvokingWorkflow(t *testing.T) {
 	for caseNumber, tt := range []struct {
-		inspectOS bool
 		reference string
 	}{
-		{inspectOS: true, reference: "uri/for/pd"},
-		{inspectOS: false, reference: "uri/for/pd"},
+		{reference: "uri/for/pd"},
```
Comment (dntczdx) [inline cli_tools/common/disk/inspect_test.go:39]: then it's one single test case, no need to loop

Comment (EricEdens) [reply cli_tools/common/disk/inspect_test.go]: done!

**Pull request merged**